### DISCUSSION TITLE
Active Care?

### DISCUSSION THREAD

**Discussion by CommodoreSchlippy** [chummer5a/chummer5a#5160] [General]
I was just curious if the Chummer5a project is still being cared for by anyone, or if its officially gone abandonware? I know coding is hard, lord knows I get a little deer-in-headlights at the sight sometimes, but I was just curious if its been moved on to another or not?

**Comment by ArchonMegalon** [comment]
Where does this come from? In the last months there were 24 commits (thanks do @DelnarErsike!)...

**Reply by CommodoreSchlippy** [reply]
I'm still kinda new to how Github works; I thought commits were like 'suggested' changes as opposed to actual changes. Ignore me then.
